sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -6,6 +6,7 @@ import java.io.BufferedReader
 import java.io.IOException
 import java.io.InputStreamReader
 import java.net.HttpURLConnection
+import java.net.HttpURLConnection.HTTP_UNAVAILABLE
```

Comment:
is this actually used? we already import `HttpURLConnection` and we reference `HTTP_UNAVAILABLE` from that import

Reply:
i think this can be removed

Reply:
Yeah I can get rid of it